### DISCUSSION TITLE
withdrawal

### DISCUSSION THREAD

**Discussion by jaycedevan** [zkSync-Community-Hub/zksync-developers#352] [Polls]
In zkSync, what are the key differences between a "fast" and a "standard" withdrawal, and under what circumstances would a developer choose to use one over the other?'

"Fast" withdrawals prioritize speed by allowing users to withdraw funds quickly but with potentially higher fees while "Standard" withdrawals offer lower fees but may take longer to process.

**Comment by casesumteryan** [comment]
In zkSync, "fast" withdrawals prioritize speed with potentially higher fees, while "standard" withdrawals offer lower fees but may take longer to process. Developers may choose "fast" withdrawals for urgent transactions and "standard" withdrawals for cost-effectiveness.

**Comment by atzipajabken** [comment]
The key difference between "fast" and "standard" withdrawals in zkSync is that "fast" withdrawals prioritize speed with potentially higher fees, while "standard" withdrawals offer lower fees but may take longer to process. Developers may choose to use one over the other depending on their priorities and the specific needs of their application.

**Comment by bxpana** [comment]
@jaycedevan where are you seeing the mention of "fast" and a "standard" withdrawals?
